AppTaskQueue/appscale/taskqueue/queue.py

```diff
@@ -125,6 +125,7 @@ def __init__(self, queue_info, app):
         self.task_retry_limit = retry_params['task_retry_limit']
 
     self.validate_config()
+    self.prepared_statements = {}
```

Comment:
Ohhh.. Sorry, I missed the fact that all these statements are string constants=( After the first look at these I was sure that they're format strings which get different parameters every time and so will be different for each call of function. Now I see that parameters of statements get its values after statement are prepared.